contracts/MyToken.sol

```diff
@@ -1,17 +1,35 @@
 pragma solidity ^0.4.24;
 
 import "openzeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+import "./DSToken.sol";
 
 contract MyToken is ERC721Token {
 
+  address daiAddress;
+
   constructor (string _name, string _symbol) public ERC721Token(_name, _symbol) {
+  }
+
 
+  function setDaiAddress(address _addr) public {
```

Comment:
I wonder if this can be set in the constructor instead of having to call a separate method.

Reply:
I think it's better to have a setter in case the contract address needs to be changed, but it can also be initialized in the constructor. My updates have restricted permissions on this function.